init

d-smyrnov opened the pull request.

d-smyrnov committed:
init

d-smyrnov committed:
Safor fix

d-smyrnov committed:
safor 2560 slider fix

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,177 @@
+            </div>
+            
+            <div class="one">
+                <h1>SAFOR</h2>
```
These two h1 should be represented as one.

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,177 @@
+                </div>
+        
+                <ul class="two_text">
+                    <li>
```
Nesting p into li makes no sense

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,177 @@
+    
+    </div>
+
+    <div class="four">
```
replace div for each section with section tag

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,177 @@
+            <h2>Install app IOS / ANDROID</h2>
+            <h2>Complete the registration</h2>
+            <h2>SECURE YOUR LIFE IN ONE TAP</h2>
+            <p>Additional</p>
```
Margin from top item looks incorrect on 1000px resolution

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,177 @@
+            <div class="head">
+                <a href="#"><img src="img/head/logo.svg"></a>
+                <ul>
+                    <a href="#"><li>How it works</li></a>
```
Incorrect font-size on 759px resolution

d-smyrnov committed:
Muha2399 requested changes

d-smyrnov committed:
Polind fixes

remove !important

d-smyrnov committed:
Polind fixes

has already been defined issues
0 shouldn't have units specified issues
and others

d-smyrnov committed:
Poling removing whitespaces

d-smyrnov committed:
Safor fixes

has already been defined fixed
whitespaces removed

d-smyrnov committed:
Safor 0 has specified units fixed

d-smyrnov committed:
Polind fixes

background blend mode issue
left+transform issues

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,160 @@
+                <ul>
+                    <li>
+                        <img src="img/five/Polygon.png">
+                        <p>Do not waste time to explain your location and nature of the threat, guard 
```
On 1024px i can see text from white block, when grey is active. Please fix.

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,160 @@
+        <section class="six">
+            <h1 class="six_headline">HAVE QUESTIONS? CONTACT US</h1>
+            <form class="six_inputs" action="#">
+                <input type="text" placeholder="NAME">
```
Distance between placeholder and bottom border should be as on mockup

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,160 @@
+            <p>© SAFOR 2018 All Rights Resevered 85 Great Portland Street, First Floor London, W1W7LT, United Kingdom</p>
+            <div>
+                <a href="#"><img src="img/footer/1.svg"></a>
+                <a href="#"><img src="img/footer/2.svg"></a>
```
On resolution 375, size of instagram icon incorrect. Also check distance between icons.

muha2399 replied:
Try to use another instagram and maybe another social icons on lower resolution.

muha2399 commented on Safor/index.html:
```diff
@@ -0,0 +1,160 @@
+                        <hr>
+                        <hr>
+                        <hr>
+                        <hr>
```
On resolution 320 these lines should be 1px, but now they are 2px

d-smyrnov committed:
Safor fixes

 fixing position of white box, icons sizes and distances

d-smyrnov committed:
Safor (placeholder padding)

d-smyrnov committed:
Safor icons fix

volo-maslow commented on Safor/index.html:
```diff
@@ -0,0 +1,160 @@
+        <title>Document</title>
+    </head>
+    <body>
+        <section class="topLayer">
```
Strange naming of the section, please use dashes or underscores to separate different words.

volo-maslow commented on Safor/index.html:
```diff
@@ -0,0 +1,160 @@
+                </ul>
+            </div>
+        </section>
+        <section class="six">
```
Change names of sections to be more descriptive, for example: footer-feedback or feedback-form.
Please do this for every section and sub div in the document.

d-smyrnov committed:
Safor titles changed
